runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -485,6 +488,15 @@ public void onChatMessage(ChatMessage event)
 			String fileName = "Collection log (" + entry + ")";
 			takeScreenshot(fileName, SD_COLLECTION_LOG);
 		}
+
+		if (config.screenshotCombatAchievements() && chatMessage.contains("combat task"))
```

Comment:
It will be cheaper to first check for the message containing a "combat task" substring than to hit the config cache on every chat message.
```suggestion
		if (chatMessage.contains("combat task") && config.screenshotCombatAchievements())
```